umbraco/Umbraco-CMS
# Netcore: Reintroduce friendly bun untestable published content extensions

### Details
- Introduce an internal service location 🤯 
- Use that service location in extension methods so views can easily be migrated.

### Test
- Use the normal extension methods that could be used in v8.
- Use the query builder to generate queries, and ensure they work.
